backend/tracim_backend/lib/cleanup/cleanup.py

```diff
@@ -251,6 +252,19 @@ def delete_workspace_agenda(self, workspace_id: int) -> typing.Optional[str]:
     def delete_user_associated_data(self, user: User) -> None:
         """deleted all stuff about user except user itself, his workspaces and user content"""
 
+        # INFO - G.M - 2021-03-22 UserFavoriteContent
+        favorites_contents = self.session.query(FavoriteContent).filter(
+            FavoriteContent.user_id == user.user_id
+        )
```

Comment:
Is it not possible to use the lib here?

Reply:
It may being possible, but there is currently not such thing as "delete all favorite" in the lib, so i don't choose to use it. In this specific case, it does also work because we do a true deletion but for some other type, we do this explicitly in cleanup as lib doesn't have "hard deletion" mecanism. We may decide to move part of these logic in specific lib. Note that cleanup is a lib too.

Reply:
OK, we can leave in this state.